pkg/operator/controller/status/controller.go

```diff
@@ -288,40 +286,50 @@ func checkAllIngressesAvailable(ingresses []operatorv1.IngressController) bool {
 
 // computeOperatorDegradedCondition computes the operator's current Degraded status state.
 func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController) configv1.ClusterOperatorStatusCondition {
-	degradedIngresses := make(map[*operatorv1.IngressController]operatorv1.OperatorCondition)
-	for i, ingress := range ingresses {
-		for j, cond := range ingress.Status.Conditions {
-			if cond.Type == operatorv1.OperatorStatusTypeDegraded && cond.Status == operatorv1.ConditionTrue {
-				degradedIngresses[&ingresses[i]] = ingress.Status.Conditions[j]
+	degradedCondition := configv1.ClusterOperatorStatusCondition{
+		Type: configv1.OperatorDegraded,
+	}
+
+	foundDefaultIngressController := false
+	for _, ic := range ingresses {
+		if ic.Name != manifests.DefaultIngressControllerName {
+			continue
+		}
+		foundDefaultIngressController = true
+		foundDegradedStatusCondition := false
+		for _, cond := range ic.Status.Conditions {
+			if cond.Type != operatorv1.OperatorStatusTypeDegraded {
+				continue
+			}
+			foundDegradedStatusCondition = true
+			switch cond.Status {
+			case operatorv1.ConditionFalse:
+				degradedCondition.Status = configv1.ConditionFalse
+				degradedCondition.Reason = "IngressNotDegraded"
+				degradedCondition.Message = "The default ingress controller reports Degraded=False."
```

Comment:
Can we use `manifests.DefaultIngressControllerName` for "default" here and in other messages?